terraform/terraform-dependencies/main.tf

```diff
@@ -15,7 +15,7 @@ resource "kustomization_resource" "training_operator" {
 # Create NFS resource
 resource "helm_release" "nfs_client_provisioner" {
   name       = var.nfs_provider_information.release_name
-  chart      = "charts/nfs-server-provisioner-1.1.3.tgz"
+  chart      = "${path.module}/../../charts/nfs-server-provisioner-1.3.1.tgz"

```

Comment:
This seems to be related to an earlier patch that has been provided in #52. Please open an issue to the problem that you encountered to make discussion easier.

Reply:
This has been addressed in #59